### PR TITLE
Fix invalid example in now-examples

### PR DESCRIPTION
**Pull request by quietshu**
Currently invalid now-examples subdir will be treated as valid, for example: https://now-example-files.zeit.sh/info.json?repo=zeit/now-examples/tree/master/404.

This PR fixed this case and responses with `{}` instead.

### CODE REVIEW

File: lib/repo-info.js [hunk 1]
```diff
@@ -63,6 +63,10 @@ const getRepoInfo = async (repo) => {
         }
       }
     }
+
+    // if the path is like `zeit/now-examples/tree/master/???`
+    // but not exist in the example list
+    return null
```
Comment (joecohens) [inline lib/repo-info.js]: I think we should log the error as in the other cases

Comment (quietshu) [reply lib/repo-info.js]: Good point!

Comment (quietshu) [reply lib/repo-info.js]: Added 🙏

**Comment by quietshu** [comment]
@joecohens :approve:?

**Comment by quietshu** [comment]
Thank you ❤️

**Pull request merged**